### PR TITLE
BZ1867205: clarification

### PR DESCRIPTION
**Pull request by skrthomas**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1867205

For doc versions 4.5+

Direct link to doc preview: https://deploy-preview-30086--osdocs.netlify.app/openshift-enterprise/latest/networking/dns-operator.html#nw-dns-forward_dns-operator

Ready for QA @lihongan

### CODE REVIEW

File: modules/nw-dns-forward.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@
 [id="nw-dns-forward_{context}"]
 = Using DNS forwarding
 
-You can use DNS forwarding to override the forwarding configuration identified in `/etc/resolv.conf` on a per-zone basis by specifying which name server should be used for a given zone.
+You can use DNS forwarding to override the forwarding configuration identified in `/etc/resolv.conf` on a per-zone basis by specifying which name server should be used for a given zone. If the forwarded zone is the Ingress domain managed by {product-title}, then the upstream name server must be authorized for the domain.
```
Comment (bmcelvee) [inline modules/nw-dns-forward.adoc:8]: Suggested edit: 
If the forwarded zone is a domain managed by {product-title}, such as the Ingress domain, then...

**Comment by bmcelvee** [comment]
Just a suggested reword, otherwise LGTM! Once the PR goes through QE and SME review, I'll be happy to merge. :sun_with_face:

**Comment by lihongan** [comment]
It looks great! Thank you @skrthomas

**Comment by bmcelvee** [comment]
@danehans I'm going ahead with the merge and cherry pick on this one since it's a straight forward update. Please let us know if any additional changes are required and we can make those in a follow-up PR. Thanks!

**Pull request merged**